<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotter</class>
 <widget class="QMainWindow" name="plotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="butClear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear plots</string>
    </property>
   </widget>
   <widget class="QwtPlot" name="qwtPlot1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QwtPlot" name="qwtPlot3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>791</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QwtPlot" name="qwtPlot2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>791</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
